docs/APIRef.Matchers.md

```diff
@@ -42,14 +43,22 @@ Find an element by text, useful for text fields, buttons.
 ```js
 await element(by.text('Tap Me'));
 ```
+
+#### `by.label(label)`
+Find an element by accessibilityLabel(iOS) or contentDescription(Android), useful for text fields, buttons.
```

Comment:
Accessibility label is useful for almost any UI element on iOS. As [per documentation](https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel?language=objc), this should be properly set to any element with custom appearance, such as images. The system helps by trying to set a value to this property be similar to the title of elements such as text fields or buttons, but also for other elements. The “useful” comment implies incorrect here.

Reply:
Ok. I'll remove the useful part.